x-pack/plugins/actions/server/types.ts

```diff
@@ -45,20 +49,28 @@ export interface ActionsConfigType {
 export interface ActionTypeExecutorOptions {
   actionId: string;
   services: Services;
+  // This will have to remain `any` until we can extend Action Executors with generics
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   config: Record<string, any>;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   secrets: Record<string, any>;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
An issue has been added to address this: https://github.com/elastic/kibana/issues/64147